### ISSUE TITLE
GitHub actions for the JSON file

### ISSUE THREAD

**Issue by kccuber-scratch** [lopste/forumoji#13]
I think it would be better if we tasked a github action with keeping the JSON file. Manually editing will get really tedious and an automated Python script would do the job of manually maintaining the JSON.
Emojis could be named as "name_codepoint" and a string split after iterating over each emoji's name could be used to get both the codepoint and name of the emoji, and a link.

**Comment by lopste** [comment]
Yes, this sounds like a really good idea. I just gotta find the API for Actions, though.

**Comment by kccuber-scratch** [comment]
Sorry, I had to reopen. Instead of making to so it automatically picks up all the emojis in the folder, we should do it so that you can create an issue (using a specific issue template) and it will automatically add the emoji(s) to the folder and the JSON file
